api/github.js

```
@@ -1,46 +1,75 @@
 }
 
+function addRateLimitHeaders(res, headers) {
+  const map = {
+    'x-ratelimit-limit':     'X-RateLimit-Limit',
+    'x-ratelimit-remaining': 'X-RateLimit-Remaining',
+    'x-ratelimit-reset':     'X-RateLimit-Reset',
+    'retry-after':           'Retry-After',
+  };
+  for (const [src, dst] of Object.entries(map)) {
+    const val = res.headers.get(src);
+    if (val != null) headers[dst] = val;
```

ESLint

[error] Expected '!==' and instead saw '!='.

(eqeqeq)